loops.js

```diff
@@ -87,5 +103,21 @@
 // etc.
 // ==========================================
 
-
+for (let i = 0; i < 100; i++) {
+
+    const dividedByThree = i % 3 === 0;
+    const dividedByFive = i % 5 === 0;
+
+    if (dividedByThree && dividedByFive) {
+        console.log("fizzbuzz");
+    } else if (dividedByThree) {
+        console.log("fizz");
+    } else if (dividedByFive) {
+        console.log("buzz");
+    /*} else if (i === 0) {    Mogelijkheid om de 0 uit te sluiten zou bovenaan de if statement moeten.
```

Comment:
Ik had dit zelf helemaal over het hoofd gezien maar je zou bij de 'if' en 'else if' erbij kunnen zetten '&& i !== 0'. Er is waarschijnlijk nog een betere manier ;)

Reply:
@Danielooo 

Ik snap niet helemaal wat je bedoelt, waar zou ik de && erbij moeten plaatsen, en wat doet dit voor mijn code?

Of heb je het over de `else if (i === 0)`
Dat ik deze eruit kan halen?